### PR TITLE
Add parallelization error message

### PR DESCRIPTION
**Pull request by maprules1000**
## Description
When a paralleized build is failed because of missing finalize calls from some pipeline we don't show the user an error.

This new error shows the user that a missing parallel build was the cause of the build failure, and tells them how many parallel builds were expected and how many were finalized.

## Pictures
screen shot 2018-07-26 at 2 21 23 pm

## Reviewing this PR
_**Is Covered by Automated Tests?: YES**_

### CODE REVIEW

File: app/templates/components/build-header.hbs [hunk 1]
```diff
@@ -79,10 +79,17 @@
       </div>
   {{else if build.isFailed}}
     {{#if (eq build.failureReason 'missing_resources')}}
+      {{#if isParallelBuildError}}
       <div class="p-2 m-2 bg-gray-000 rounded border border-gray-100">
-        <div class="font-semibold mb-1">Your CI service or client failed to correctly upload build and snapshot resources.</div>
-        This build can be safely ignored. It will not be used as a baseline for visual diffs. If these failures happen frequently, check your CI or client logs for failures.
+        <div class="font-semibold mb-1">Percy expected {{parallelBuildsExpected}} finalize calls for your parallelized build but only receievd {{parallelBuildsReceived}} finalize call(s).</div>
+        Check your CI logs to see if there was an error in one of your build pipelines. Make sure your parallel builds are <a href="https://docs.percy.io/docs/parallel-test-suites" target="_blank">configured properly</a>. If you'd like help troubleshooting this, feel free to <a href="#" {{action showSupport}} data-test-build-overview-show-support>reach out</a> and we can help get this fixed.
```
[the remark this answers is not in the document]
Comment (cadeParade) [reply app/templates/components/build-header.hbs]: There is a `pluralize` helper you could use for the `call` word here.

**Comment by timhaines** [comment]
@maprules1000 I’d like to suggest rewording: 
```
Percy expected 2 finalized calls for your parallelized build but only received 1 finalize call
```
To:
```
Only 2 of the expected 5 parallelized build processes reported they were finished.
```
I’m suggesting this because our customers generally don’t know what a finalize call is, unless they’ve gone poking around in our SDK code.

File: app/templates/components/build-header.hbs (hunk 1 above)
[the remark this answers is not in the document]
Comment (cadeParade) [reply app/templates/components/build-header.hbs]: The goal of these changes is to lessen support burden, but let's leave the support button in for now and see what sorts of different requests we get. Then we can update further to adapt to the new queries. WDYT?

File: tests/integration/components/build-header-test.js [hunk 2]
```diff
@@ -64,4 +65,20 @@ describe('Integration: BuildHeader', function() {
     this.$('[data-test-build-overview-show-support]').click();
     expect(showSupportStub).to.have.been.called; // eslint-disable-line
   });
+
+  it('sends showSupport action when clicking "reach out" when missing parallel builds', function() {
```
[the remark this answers is not in the document]
Comment (cadeParade) [reply tests/integration/components/build-header-test.js]: A Percy snapshot is taken above of this state, so this seems fine.
